### ISSUE TITLE
Plan name under the selected plan details under plan details tab - AB-127

### ISSUE THREAD

**Issue by GoogleCodeExporter** [willdwyer/bcbsmaissuestracker#183]
```
What steps will reproduce the problem?
1. Log on to PClient5 as a BCBSMA Administrator
2. Click on Manage Plan
 Enter search criteria:
 Effective Date - 2015
 Financial Arrangement - ASC
 Market Segment - Large Group 100+
 Plan Type - Standard
 Click - Show Me Plans"

3. Select 
600353HR_Network_Blue_NE_Options_Ded_v.4_80-0220LGASC1-1-15 and click create 
new plan from this plan and select custom next 
4.When the Plan details tab loads notice the plan name under the selected plan 
details section below


Actual Result:
It says copy(7) of 600353HR_Network_Blue_NE_Options_D...


Expected Result:
Per requirements it is supposed to indicate the base standard plan name which 
is 600353HR_Network_Blue_NE_Options_Ded_v.4_80-0220LGASC1-1-15.

Is the system creating the custom  out of the copy number 7 of the standard 
plan design ?


Test Case #
E2E_14
```

Original issue reported on code.google.com by `g4gaur...@gmail.com` on 11 Jun 2015 at 3:25

Attachments:
- [PlanName.JPG](https://storage.googleapis.com/google-code-attachments/bcbsmaissuestracker/issue-183/comment-0/PlanName.JPG)

**Comment by g4gaurang** [comment]
Workaround suggested above is not helpful and efficient. It is an additional step and time in the process as it entails going back to the plan master just to view the plan design name. 

Keep us posted about a fix.

**Comment by TimHeasley** [comment]
AB-127 and AB-146 are in queue to be picked up following the closure of 2 other issues on the employer enclave. In priority ranking, AB-127 will be picked up first, followed by AB-146. This may stretch into Sprint 6 based on capacity and timing of the 2 other bugs resolution (Sprint 5: 7/29-8/11, Sprint 6: 8/12-8/26).

**Comment by TimHeasley** [comment]
Update for this issue: 

Due to a higher priority issues on the employer enclave, these have not been picked up yet.  They are first in line pending the resolution of the production bugs that are currently in progress with development.

Currently, it is not anticipated that they will begin before the next sprint (Sprint 6), which starts next Wednesday.

**Comment by willdwyer** [comment]
Hi Tim,

Any update on this?

**Comment by TimHeasley** [comment]
Will, 
Sprint 6 is starting today and goes through August 26th. I will reach out to the product team to see if they can give us any updates.  Thanks

**Comment by TimHeasley** [comment]
Will, we received the following information from the product team - "Based on a brief update yesterday, there is a one remaining blocker item that must be completed prior to moving on to the next up item on the list (AB-127).  When it is picked up by the developer, you will be notified."

**Comment by TimHeasley** [comment]
Updating details here with information that was added to AB-146

"Will, I have contacted the product team regarding this issue AB-146 (and issue AB-127). The product team just resolved a blocker bug today that had precedence over these 2 items. I will continue to work with the product team to try and get a firm timeframe. As soon as I know the details I will add them here."

**Comment by willdwyer** [comment]
OK.  Thanks, Tim.

**Comment by markderg** [comment]
Update from Product Team - Items 221 and 183 are scheduled to be completed as part of this Product Sprint.  The completion for these 2 issues is scheduled for 9/8.  **NOTE:  This is provided that no production issues take priority during this period.  Pending the completion of these 2 issues, we will need to account for internal and external testing requiring these code code changes loaded into our test environment and then PClient 5.  Based on the development and testing timeline we will plan on deployment the end of September (9/29) with a tentative timeline below.  
• 9/8 – Completion of Development
• 9/9 – Deploy code for internal testing
• 9/10, 9/11, 9/14 – Services Testing in CPE2
• 9/15 – Deployment to PClient 5
• 9/16 – Services Testing in PClient 5
• 9/18 – 9/22 -  BCBSMA Testing in PClient 5
• 9/23 – Go/No Go (BCBSMA and HR)
• 9/24 & 9/25 – HR QA Validation of Code as part of Deployment to Production
• 9/28 – Deployment to Production
• 9/29 – Post Production Checkout by BCBSMA

**Comment by TimHeasley** [comment]
Tested and confirmed in PClient5 that plan name appears correctly in the plan details section.  
Ready for retest,

**Comment by willdwyer** [comment]
Hello,

This specific defect appears to be fixed.  However, note 2 issues we encountered.  Per screenshots, 1. there is a junk #VALUE! appearing in header, and 2. there are 2 underscores appearing between HR job # and plan name on header.

Please move to production.

![screen shot 2015-09-17 at 10 02 56 am](https://cloud.githubusercontent.com/assets/13453008/9935365/30d0d520-5d24-11e5-8e62-e9a6795ddd17.png)
![screen shot 2015-09-17 at 9 46 50 am](https://cloud.githubusercontent.com/assets/13453008/9935366/30d1f48c-5d24-11e5-9b5a-be98b06b82a7.png)

**Comment by michparrish** [comment]
Great, thank you for testing. Glad it was a success.
To the 2 issues above:

#1 - you didn't choose a Preferred Account Name yet - that's what would fill in in between the 2 underscores
#2 - you chose a 2015 plan design and this plan's structure was incorrect. Somehow, there were a few plans that had the 'General' header and the 'SBC File Output (Optional)' field flipped (we've ensured there are no 2016 plan designs with this issue). When this happens, the field that the workbooks are looking for isn't available so it throws the #VALUE! value. We updated that structure in pclient5 for this plan if you want to take a look. Additionally, we can update this in Prod but we will need your approval. It's a quick change that will not impact anything else. However, we don't need to do this b/c you aren't going to be generating 2015 SBCs with the tool anyway, due to the difference in structure between the 2015 and 2016 plan designs (adding the new government regulated website field to the 2016 plans). So please let me know if you'd like us to make this small change to the 2015 plan in Prod or leave it as is.

We are going to look to deploy the code by the end of next week. More to come from Mark in an official email.

Thanks, Michelle

**Comment by markderg** [comment]
Code Deployment Schedule - See below.  Looking to Promote Code Fix to PROD on 9/25 at 10 pm

Reason for Deployment:  Fix 2 defects:  1).  Header from previous custom plan generated remains on header -#221/Issue AB 146 in JIRA & 2).  Plan name under the selected plan details tab - #183/Issue AB 127 In JIRA
Deployment Tasks
HR Testing in CPE2:  Tim Heasley
HR Testing Approval Date in CPE2:  9/15
HR Testing in PClient 5:  Tim Heasley
HR Testing Approval Date in PClient 5:  9/16
BCBSMA Testing in PClient 5:  Will Dwyer
BCBSMA Testing Approval Date in PClient and Promotion to PROD:  9/17
HR Approvers of Code Deployment to PROD:  Mark Der Garabedian, Michelle Parrish, Kris Kennedy, and Judy Li
HR Approval Date:  9/17
Timeline for HR QA Testing:  9/23 and 9/24
Timeline for Deployment:  9/25 at 10 pm
Timeline for BCBSMA Post Production Checkout:  9/28

**Comment by markderg** [comment]
[P19 5 11 29 Code Deployment 9.25.15 V2.xlsx](https://github.com/g4gaurang/bcbsmaissuestracker/files/2627/P19.5.11.29.Code.Deployment.9.25.15.V2.xlsx)

Final deployment checklist attached with all required approvals and tasks.

**Comment by willdwyer** [comment]
Hello,

This defect appears to be fixed, and is now being closed.
